### DISCUSSION TITLE
How to use GROUP Share Level without Hadoop

### DISCUSSION THREAD

**Discussion by A-little-bit-of-data** [apache/kyuubi#6701] [Q&A] [answered]
I started the kyuubi (v1.9.1) pod in a k8s environment and used spark SQL to query S3 data. There is no Hadoop environment at all. However, according to the documentation, the kyuubi.session.group.provider configuration seems to only have hadoop. Does that mean that GROUP Share Level can only be used in an environment with hadoop?

**Comment by wangzhigang1999** [comment, marked as answer]
According the code, it appears that Hadoop is currently the only built-in implementation. 

However, according to **org.apache.kyuubi.plugin.PluginLoader#loadGroupProvider**, you can implement the **GroupProvider** interface to create your own plugin.
